scriptworker/constants.py

```diff
@@ -22,6 +23,17 @@
     'intermittent-task': 7,
 }
 
+_ALLOWED_MOBILE_GITHUB_OWNERS_PATTERN = '|'.join([
```

Comment:
IMHO, we should have this data in its raw-est format up here: when we're adding new users, this is the only place we want to have to change. We should move the regex-y bit (escaping, joining with `|`) down to where it's needed.
Then, if we are using these names down below in a non-regex-y way, we don't need to parse out the `|` :smile:

Reply:
True. That variable dated back before I turned the mobile regexes into a list-comprehension. It doesn't provide much value anymore. I implemented your suggestion. Thank for spotting it!